tools/chpldoc/chpldoc.cpp

```diff
@@ -320,6 +320,18 @@ static bool symbolNameBeginsWithChpl(const Decl* node) {
   return false;
 }
 
+static bool hasAttribute(const AstNode* n, UniqueString attrName) {
```

Comment:
This is probably a question for someone on the dyno subteam, but I'm surprised this isn't normally defined in whatever file handles attributes.  I'd be inclined to move it there.